service-catalog/systems/grafana.yaml

```diff
@@ -2,7 +2,15 @@ apiVersion: backstage.io/v1alpha1
 kind: System
 metadata:
   name: grafana
-  description: Grafana system
+  title: Grafana (private)
+  description: Grafana is a multi-platform open source analytics and interactive visualization web application. This instance is available to community members only.
+  annotations:
+    operate-first.cloud/featured: "true"
+    operate-first.cloud/logo-url: https://raw.githubusercontent.com/grafana/grafana/main/public/img/grafana_icon.svg
+  links:
+    - url: https://grafana.operate-first.cloud/
```

Comment:
Maybe we should remove this link since its for the public instance?

Reply:
No, this is link to the private instance. Public instance has https://grafana-public.operate-first.cloud/ url

Reply:
Sorry about that, I should check more carefully next time!